### PR TITLE
Feature/changes to components

### PR DESCRIPTION
**Pull request by alejandroVaron**
What:  Fix changes to components

Why:  The application requires: 
- Change the color inputs in the changeEvent component
- Fix the function of calculate minutes 
- Fix the routes of components

Color event
Calculate minute

### CODE REVIEW

File: front-end-yaz/src/components/Calendar.vue [hunk 1]
```diff
@@ -171,29 +184,75 @@ export default {
         this.lastDate = nativeEvent.date;
         this.startTime = nativeEvent.time;
         // ----------
-        var end = this.calculateHour(nativeEvent.time);
-        var mayor = 0;
+        var start = this.calculateMinute(nativeEvent.time);
+        if (start.substring(3, 5) == "0") {
+          start = start + "0";
+        } else {
```
Comment (juanDuqueS) [inline front-end-yaz/src/components/Calendar.vue:190]: this else does not do anything

File: front-end-yaz/src/components/Register.vue [hunk 2]
```diff
@@ -45,16 +45,20 @@
                 <v-select
                   v-model="select"
                   :items="items"
-                  :rules="[v => !!v || 'Item is required']"
+                  :rules="[(v) => !!v || 'Item is required']"
```
Comment (juanDuqueS) [inline front-end-yaz/src/components/Register.vue:48]: what is v?

**Review by juanDuqueS — APPROVED** [review]
subtle things to change but nice work nonetheless

**Review by prometteo14 — APPROVED** [review]

File: front-end-yaz/src/components/Calendar.vue (hunk 1 above)
Comment (alejandroVaron) [reply front-end-yaz/src/components/Calendar.vue:190]: yes, i will fix

File: front-end-yaz/src/components/Register.vue (hunk 2 above)
Comment (alejandroVaron) [reply front-end-yaz/src/components/Register.vue:48]: v is "validation",  is the props of the v-select

**Pull request merged**